example/src/main/resources/templates/webeid-login.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Signing you in…</title>
+</head>
+<body>
+<div id="error-message" class="alert alert-danger" style="display: none;" role="alert">
+    <div class="message"></div>
+    <pre class="details"></pre>
+</div>
+
+<script type="module" th:inline="javascript">
+    import { showErrorMessage } from "/js/errors.js";
+    (function () {
+        const frag = location.hash ? location.hash.substring(1) : "";
+        if (!frag) { showErrorMessage({ code: "UNKNOWN_ERROR", message: "Missing authentication payload" }); return; }
+
+        let payload;
+        try { payload = JSON.parse(atob(frag)); } catch (e) {
+            console.error("Failed to parse payload", e);
+            showErrorMessage({ code: "UNKNOWN_ERROR", message: "Failed to parse authentication payload" });
+            return;
+        }
+
+        if (payload["error"]) {
+            showErrorMessage({
+                code: payload["code"] ?? "UNKNOWN_ERROR",
+                message: payload["message"] ?? "Authentication failed"
+            });
+            return;
+        }
+
+        const authToken = payload["auth-token"];
+
+        fetch(/*[[${loginProcessingPath}]]*/, {
```

Comment:
We have used `async` / `await` keywords elsewhere, maybe let's use them here too for consistency:


```suggestion
        try {
            const res = await fetch(/*[[${loginProcessingPath}]]*/,
            ...
        } catch (error) {
            showErrorMessage(error);
        }
```